ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -309,6 +311,16 @@ internal class CustomerCenterViewModelImpl(
                 }
             }
 
+            CustomerCenterConfigData.HelpPath.PathType.CUSTOM_ACTION -> {
```

Comment:
Just to match the other cases in the when:

```suggestion
            HelpPath.PathType.CUSTOM_ACTION -> {
```